ChangeLog

```diff
@@ -3,12 +3,106 @@ Changelog
 
 ## v2.4.0
 
-- Enum: uniq is modified to use structural comparison internally, instead
-    of previously used physical comparison, which was most likely a bug.
-    The old behavior is provided by the new Enum.uniqq function.
-    Enum.uniq_by was added for additional convenience.
+- ignore deprecation warnings
+  (Gabriel Scherer)
+- BatBitSet: use Bytes instead of String
```

Comment:
merge with the "deprecation warnings" and "corrected many warnings" entries (listing both authors), with some more general description like "update codebase wrt. new OCaml warnings -- preparation for future `-safe-string` compat".